packages/console/src/hooks/use-user-preferences.ts

```diff
@@ -35,10 +36,19 @@ const useUserPreferences = () => {
   const { isAuthenticated, error: authError } = useLogto();
   const userId = useLogtoUserId();
   const shouldFetch = isAuthenticated && !authError && userId;
+  const api = useStaticApi({ prefixUrl: adminTenantEndpoint, resourceIndicator: meApi.indicator });
```

Comment:
nit: Could extract a useAdminApi (or useSystemApi) hook. Embed the adminTenantEndpoint prefix.  Most likely there will only be two types of API calls,  useApi and useAdminApi.

Reply:
yes, since this hook is the only place to use i think it's fine for now. maybe add that when @charIeszhao or @xiaoyijun  need another hook to perform Admin Tenant APIs?